pages/vi/vi-vagrant.md

```diff
@@ -158,6 +158,12 @@ For more information go [here](http://stackoverflow.com/questions/23874260/error
 
 2. When trying to access http://127.0.0.1:5985/apps/_design/bell/MyApp/index.html in step 4, you may experience an error such as the following: "no_db_found". A simple solution will be using ```vagrant destroy``` to delete the current machine, then try ```vagrant up``` to rebuild it.
 
+3. When you run ```vagrant up``` from command prompt on windows, you might get the following error :
+<blockquote>The executable 'curl' Vagrant is trying to run was not
+found in the %PATH% variable. This is an error. Please verify
+this software is installed and on the path.</blockquote> A simple solution is to add Cygwin bin folder to path variable or use Git Bash rather than command prompt to run vagrant up.
+For more information visit [issue](https://github.com/hashicorp/vagrant/issues/6788)
```

Comment:
Please use Markdown instead of HTML and ensure that you are following writing conventions!  
You should add the three tick marks before and after "vagrant up" to highlight it.

For consistency, consider just using quotation marks instead of a block quote.

Reply:
@thesiege208 Sorry for the HTML. Fixed it. Please review the same